cluster/src/main/java/com/hartwig/pipeline/cram/CramConversion.java

```diff
@@ -66,10 +71,18 @@ public CramOutput output(SingleSampleRunMetadata metadata, PipelineStatus jobSta
         String cram = new File(outputCram).getName();
         String crai = CramOutput.craiFile(cram);
         Folder folder = Folder.from(metadata);
+
+        String fullCram = format("%s%s/%s", folder.name(), NAMESPACE, cram);
+        String fullCrai = format("%s%s/%s", folder.name(), NAMESPACE, crai);
+
+        AdditionalApiCalls.instance().register(fullCram, new LinkFileToSample(metadata.sampleId()));
```

Comment:
singleton anti-pattern. makes the class impossible to unit test. let's invert the dependency and make AdditionalApiCalls a field.

Also I was thinking we'd use the "ReportComponents" for this rather than introducing a new class in which to register them.

Reply:
Meant to mention - this is absolutely just a waypoint, it won't be staying that way, working on getting rid of it now.

I was sitting on the fence with putting it in the `ReportComponents`, wasn't sure about overloading but am happy to make that change.